### ISSUE TITLE
Wrong eval sign when playing Black

### ISSUE THREAD

**Issue by tpoppins** [SamuraiDangyo/Sapeli#2]
Sapeli 1.01 doesn't follow UCI protocol [specs](http://download.shredderchess.com/div/uci.zip) when playing Black. The specs state:

```
* score
		* cp <x>
			the score from the engine's point of view in centipawns.
```

while Sapeli always reports score from White's point of view.

Sample game.

```
[Event "?"]
[Site "?"]
[Date "2019.04.24"]
[Round "?"]
[White "__Thinker"]
[Black "Sapeli 1.01 64-bit"]
[Result "1-0"]
[FEN "7R/5K2/8/5k2/8/8/8/8 w - - 0 1"]
[GameDuration "00:00:47"]
[GameEndTime "2019-04-24T04:57:05.437 Eastern Daylight Time"]
[GameStartTime "2019-04-24T04:56:17.860 Eastern Daylight Time"]
[PlyCount "23"]
[SetUp "1"]
[TimeControl "40/120"]

1. Re8 {3.9s} Kg5 {+7.73/14 4.0s} 2. Re5+ {7.8s} Kg4 {+7.73/14 3.9s}
3. Kf6 {7.1s} Kf4 {+7.73/14 3.7s} 4. Re8 {3.9s} Kg3 {+7.80/13 3.6s}
5. Kg5 {1.9s} Kf3 {+8.13/15 3.5s} 6. Re7 {1.8s} Kg3 {+419.43/15 1.9s} 7. Rf7
Kh3 {+419.43/11 0.042s} 8. Kf4 Kg2 {+419.43/9 0.005s} 9. Kg4
Kh2 {+419.43/7 0.001s} 10. Kf3 Kh1 {+419.43/5 0s} 11. Kf2 Kh2
12. Rh7# {White mates} 1-0


```

Sapeli, playing Black, displays positive eval while being mated. It should display negative values in such cases.

This breaks GUI adjudication. E.g. when Sapeli is winning with black pieces it will display negative eval, which will eventually result in Sapeli being awarded a loss by adjudication.

**Comment by SamuraiDangyo** [comment]
Thanks for reporting this bug!
I'm not sure if this is a feature or a bug. Sapeli only reports absolute scores.
But I think I fixed it in the 1.02 branch.
https://github.com/SamuraiDangyo/Sapeli/tree/1.02

Regards

**Comment by tpoppins** [comment]
Fixed, thank you.
